# lxd-to-incus assumes the rc-service name is incus, where we use incusd
--- a/cmd/lxd-to-incus/targets_openrc.go
+++ b/cmd/lxd-to-incus/targets_openrc.go
@@ -15,17 +15,17 @@
 		return false
 	}
 
-	_, err := subprocess.RunCommand("rc-service", "--exists", "incus")
+	_, err := subprocess.RunCommand("rc-service", "--exists", "incusd")
 	return err == nil
 }
 
 func (s *targetOpenRC) stop() error {
-	_, err := subprocess.RunCommand("rc-service", "incus", "stop")
+	_, err := subprocess.RunCommand("rc-service", "incusd", "stop")
 	return err
 }
 
 func (s *targetOpenRC) start() error {
-	_, err := subprocess.RunCommand("rc-service", "incus", "start")
+	_, err := subprocess.RunCommand("rc-service", "incusd", "start")
 	if err != nil {
 		return err
 	}
